src/onefetch/cli.rs

```diff
@@ -115,6 +116,15 @@ impl Cli {
                 .possible_values(&possible_backends)
                 .help("Which image BACKEND to use."),
         )
+        .arg(
+            Arg::with_name("image-colors")
+                .long("image-colors")
+                .value_name("NUM")
+                .takes_value(true)
+                .max_values(1)
+                .default_value("16")
+                .help("NUM of colors (16-256) used in image backend."),
```

Comment:
You might want to use [`Arg::possible_values`](https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.possible_values) to limit what numbers can be used.